File: backend/backend/src/main/java/com/backend/backend/Communication/Request/UpdateUserRequest.java
package com.backend.backend.Communication.Request;

import javax.validation.constraints.NotBlank;
import javax.validation.constraints.NotNull;

public class UpdateUserRequest {

    @NotNull
    @NotBlank
    private String email;

    private String password;

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email= email;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }
}

File: ClientApp/app/SzambiztHfApplibrary/src/main/java/com/e/szambizthfapplibrary/network/Request/UpdateUserRequest.kt
package com.e.szambizthfapplibrary.network.Request


class UpdateUserRequest {

    private lateinit var email: String

    private lateinit var password: String

    fun getEmail(): String {
        return email
    }

    fun setEmail(email: String) {
        this.email = email
    }

    fun getPassword(): String {
        return password
    }

    fun setPassword(password: String) {
        this.password = password
    }

    constructor(email: String, password: String) {
        this.email = email
        this.password = password
    }
}